### PR TITLE
IndexScrollbar: Mixed HTML & JS

### PR DESCRIPTION
**Pull request by lslachciak**
[Problem] Code snippets with examples have mixed HTML markups and JS code,
 it makes that the examples are unreadable for users.

[Solution] Separation into two blocks improves readability.

Signed-off-by: Tomasz Lukawski <t.lukawski@samsung.com>

### CODE REVIEW

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/597

File: docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_indexscrollbar.htm [hunk 1]
```diff
@@ -100,7 +102,11 @@ <h2><a id='manual-constructor0.6990520032122731'></a>How to create index scroll
          &lt;/ul&gt;
       &lt;/div&gt;
    &lt;/section&gt;
-   &lt;script&gt;
+</pre>
+
+<p>JS code:</p>
+
+<pre class="prettyprint">
```
Comment (mijin85cho) [inline docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_indexscrollbar.htm]: Related with this fix, don't we need to remove `&lt;/script&gt;` on line 165, too?

Also it looks like this "JS code" section is showing part of the whole code.
If so, what about `&lt;/div&gt;` on line 166?
It is closing of `<div id="pageIndexScrollbar" class="ui-page">` in line 78, I think.

Comment (TomaszLukawskiSam) [reply docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_indexscrollbar.htm]: Yes, you are right... I will do fix. Thank you for review and attention.

File: docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_indexscrollbar.htm [hunk 2]
```diff
@@ -54,8 +54,11 @@ <h2><a id='manual-constructor0.6990520032122731'></a>How to create index scroll
 <pre class="prettyprint">
```
Comment (mijin85cho) [inline docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_indexscrollbar.htm]: Global : Would you please check the existence of "\<p>HTML code:\</p>"?

Comment (dorlinski) [reply docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_indexscrollbar.htm]: Fixed

**Pull request merged**